### ISSUE TITLE
The test result is all black

### ISSUE THREAD

**Issue by vivibyl** [tsingqguo/misf#2]
Hi, I tried to run CelebA dataset based on the pretrained model. However, the test result I got was all black :
![image](https://user-images.githubusercontent.com/96947061/166128658-a4408019-1114-407b-9330-536516647f89.png)

 Looking forward to your reply,Thanks!

**Comment by Li-Xiaoguang** [comment]
Hello, 
Thanks for your interest in our work. The pre-trained model may not load successfully.  Please check whether the program print "Loading InpaintingModel generator..."? If the pre-trained model is loaded, it will print that message.

**Comment by vivibyl** [comment]
Hello,
Thanks for your reply. For some of the reasons you gave, I did the corresponding checks and failed to solve the test problem.
1.
![b47f8e555c4e2ceb1ca16c1bfdb409a](https://user-images.githubusercontent.com/96947061/172510385-0ca67409-dab2-4b0b-bf90-e85c519c6cf3.png)

2.The console outputs the evaluation indicators：
![8b35b5cd743ddd701992dfa76410f32](https://user-images.githubusercontent.com/96947061/172510469-fb0fd463-d2b5-446d-9630-3f44cf86840a.png)
The console outputs the evaluation indicators.The following image is the path file of the test image：
![0a1c04061ad453c89ce3196cd3ce0e4](https://user-images.githubusercontent.com/96947061/172511198-9ee2a30e-ab1e-47af-9526-3c33b06b02a8.png)

3.Test Results
![64569e1e7ac0a78813f000956444dea](https://user-images.githubusercontent.com/96947061/172511379-90b06ec1-a2b0-48f5-97b0-ad4e0583ed92.png)

Looking forward to your reply,Thanks!

**Comment by Li-Xiaoguang** [comment]
Hello,
The pre-trained model was loaded successfully. It looks like something is wrong when loading the image. Please check the code for load data, especially for the loading masks.

**Comment by vivibyl** [comment]
Hello,
Thank you very much for your reply. According to your tips, I solved this problem perfectly. It is indeed a problem with loading the mask.

**Comment by codinglin** [comment]
> 您好， 非常感谢您的回复。根据您的提示，我完美地解决了这个问题。确实是加载面具的问题。

您好，能向你询问一下partial convolution dataset作为外部数据集，怎么进行设置，我的gt与in都是好的，但是pre全都是灰色，这是怎么回事
